cli/cmd/uninstall.go

```diff
@@ -57,6 +60,17 @@ and rollback any metadata changes made to your objects.`,
 				}
 			}
 
+			// delete all sources, and wait for the pods to rollout without instrumentation
+			// this is done before the instrumentor is removed, to ensure that the instrumentation is removed
+			err := removeAllSources(ctx, client)
+			if err != nil {
+				fmt.Printf("\033[31mERROR\033[0m Failed to remove all sources: %s\n", err)
+				os.Exit(1)
```

Comment:
`removeAllSources` handles some errors internally, in this case the installation will continue and remove instrumentor which might lead to stale sources, no?